.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "10"
+  - "11"
+  - "12"
```

Comment:
Suggesting `lts/*` and `node` as the versions. The first is latest LTS, the second latest stable at the point in time the test is started (so we don't have to update the versions manually).